lib/miq_pglogical/connection_handling.rb

```diff
@@ -15,8 +15,10 @@ def with_connection_error_handling
       end
 
       def pglogical(refresh = false)
```

Comment:
Also, if we are killing the caching, do we want to change the method signature to not have the method args, since they now do nothing?

I actually am kinda against this idea, since we probably would have to update all the callers, but thought I would at least bring it up as a discussion point for this review.  That said, I think it is more of a reason to include the previous implementation as part of the comment to help explain why `refresh = false` exists.

Reply:
Yes, yes, yes.    I can leave the old refresh logic commented if that helps describe the old code.

Reply:
Fixed.